documentation/asciidoc/computers/config_txt/common.adoc

```diff
@@ -0,0 +1,39 @@
+== Common Options
+
+=== Common Display Options
+
+==== `disable_overscan`
+
+Set `disable_overscan` to `1` to disable the default values of xref:configuration.adoc#underscan[overscan] that is set by the firmware. The default value of overscan for the left, right, top, and bottom edges is `48` for HD CEA modes, `32` for SD CEA modes, and `0` for DMT modes. The default value for `disable_overscan` is `0`.
```

Comment:
Maybe put default value first in the list and explain what that does, then talk about what the other option is?